index.html

```diff
@@ -21,6 +21,19 @@
       </div>
     </header>
     <!-- END: header by Daniel -->
+    <!-- Starting Section 1 by Moamin-->
+    <section class="section 1">
+      <div class="section1 text">
+        <h2>Our Program</h2>
+        <p> Our <span class="colored"> 8-month </span> program is divided into <span class="colored">12 modules </span>. The last module is
+          a <span class="colored">6-weeks intensive Final Project </span> where students work on a real project 
+          with a real partner. Our training takes place at <span class="colored">BeCentral </span> (Brussels)  
+          every <span class="colored">Sunday </span>. During the week, students are supported online in 
+          their <span class="colored">self-study </span>, homework and peer learning. </p>
+      </div>
+      <a class="section1Btn" href ="/program.html">discover more &#8594; </a>
```

Comment:
GitHub Pages is very picky with links, and although normally it is correct in a browser, I think it only works correctly without using the / character, at least before folders..so for example /img/logo.svg does not work, but img/logo.svg does 😄

Reply:
last commit is on adding section 2 image aside to section 1 contect

Reply:
> GitHub Pages is very picky with links, and although normally it is correct in a browser, I think it only works correctly without using the / character, at least before folders..so for example /img/logo.svg does not work, but img/logo.svg does smile

yeah may be, i didn't pay attention to github links thing specially with adding the '/' to the links or images :grin: